src/App.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link, Redirect, Route, Switch, withRouter } from 'react-router-dom';
+
+import { TabsPage } from './TabsPage';
+
+import './App.scss';
+
+/*
+import { RouteComponentProps } from 'react-router-dom';
+
+type TabsPageProps = React.FC<RouteComponentProps<{ tabId: string }>>;
+const TabsPage: TabsPageProps = ({ match }) => {...};
+
+or
+
+import { useParams } from 'react-router-dom';
+
+const TabsPage = () => {
+  const { tabId } = useParams<{ tabId: string }>();
+  ...
+};
+*/
+
+export const App = withRouter(
+  ({ history }) => {
+    const select = history.location.pathname;
+
+    return (
+      <>
+        <header className="header">
+          <nav className="nav">
+            <Link
+              className={
+                `nav-link
+                ${select === '/' && "nav-link-active"}`
+              }
+              to="/"
+            >
+              <h2>Home</h2>
+            </Link>
+            <Link
+              className={
+                `nav-link
+                ${select.includes('/tabs') && "nav-link-active"}`
+              }
+              to="/tabs"
+            >
```

Comment:
use NavLink instead of Link, it has activeClassName property, so you don't need to determine if the link is active by yourself
https://reactrouter.com/web/api/NavLink/activeclassname-string